Extending/Packages/Creating-a-Package/index.md

```diff
@@ -0,0 +1,146 @@
+---
+versionFrom: 8.0.0
+---
+
+# Creating a Package
+
+:::note
+This tutorial is for Umbraco 8 - however a lot of the things covered here will be the same or similar in Umbraco 7.
+:::
+
+The goal of this tutorial will be to take something that extends Umbraco and create it as a package. The tutorials starting point will be creating a package out of the dashboard from the [Creating a Custom Dashboard tutorial](../Creating-a-Custom-Dashboard/index.md), however, the process will be the same for most packages so feel free to follow along with something else!
+
+## Creating a package in the backoffice
+
+If you have something you want to make into a package, you can do so through the backoffice of Umbraco. First go to the `Packages` section and then click `Created` in the top right corner. Next you can click the `Create package` button.
+
+![Buttons to select for creating a package in the backoffice](images/creating-package-menu.png)
+
+On the `Create package` page there are 4 sections with different info to fill out. Some of the info is mandatory, most of it is not. First of all you should give the package a name at the top - we will call our dashboard the same as in the [tutorial](../Creating-a-Custom-Dashboard/index.md): `Custom Welcome Dashboard`.
+
+### The Package Properties section
+
+This section contains mostly meta data about the package and the creator. We will fill in the following things:
+
+| Property | Value | Note |
+| -------- | ----- | ---- |
+| Url | https://umbraco.com | This url will be shown as the "packages url" when others install it. Will mostly be a Github repo or similar |
+| Version | 1.0.0 | This is automatically set to 1.0.0 but can be changed or increased manually if needed |
+| Icon Url | _Blank_ | Not a mandatory value, but will appear as the package icon different places in the backoffice if set |
+| Umbraco version | 8.2.0 | It will automatically select the version your Umbraco is currently on. This is then set as the compatible version for the package |
```

Comment:
```suggestion
| Umbraco version | 8.2.0 | It will automatically select the Umbraco version you are currently using. This is then set as the compatible version for the package |
```